packages/manager/apps/dedicated/client/app/app.module.js

```diff
@@ -90,6 +90,8 @@ import chartjs from 'angular-chart.js';
 
 import moduleExchange from '@ovh-ux/manager-exchange';
 import ovhManagerFilters from '@ovh-ux/manager-filters';
+// Import Hub components for rbx1 eol banner
+import ovhManagerHub from '@ovh-ux/manager-hub';
```

Comment:
**nitpick(blocking):** We cannot import Hub into dedicated. Please find a more suitable place to define the component.